javascript/node/selenium-webdriver/firefox/index.js

```diff
@@ -421,13 +470,46 @@ class Driver extends webdriver.WebDriver {
    */
   setFileDetector() {
   }
+
+  /**
+   * Get the context that is currently in effect.
+   *
+   * @return {Context} Current context.
+   */
+  get context() {
```

Comment:
style nit: I'd prefer actual `getContext()` and `setContext(ctx)` methods since they involve RPCs (this would also be consistent with the rest of the API)